assets/js/playlist-functions.js

```diff
@@ -0,0 +1,28 @@
+/*  Ajax Create Playlist and it's form validation */
+$(document).on('submit','#CreatePlaylist', function(event){
+      event.preventDefault();
+          $.ajax({
+            method: 'POST',
+            url:create_playlist_url,
+            data: $(this).serialize()
+          }).done(function(response){
+            //Check if there is a form returned from the JsonResponse
+            if (response.form == null){
+               APPEND_TO_PLAYLISTS = "<li><a href ='" + response.view_url + "'>"
```

Comment:
Where did you get this format? ALL_CAPS_SNAKE_CASE. give me the link of the source of this guide and i'll report it for being so BAD